cmd/litcli/status.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"context"
+
+	"github.com/lightninglabs/lightning-terminal/litrpc"
+	"github.com/urfave/cli"
+)
+
+var statusCommands = []cli.Command{
+	{
+		Name:     "status",
+		Usage:    "info about litd status",
+		Category: "Status",
+		Action:   getStatus,
+	},
+}
+
+func getStatus(ctx *cli.Context) error {
+	clientConn, cleanup, err := connectClient(ctx, true)
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+	client := litrpc.NewStatusClient(clientConn)
+
+	ctxb := context.Background()
+	resp, err := client.SubServerStatus(ctxb, &litrpc.SubServerStatusReq{})
```

Comment:
I wonder if we also should try calling `lnd`'s `State.GetState` RPC here and print that as well? Just as a nice addition for CLI users.